package.json

```diff
@@ -30,7 +31,8 @@
     "nodemon": "^2.0.4",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "webpack": "^4.44.2"
+    "webpack": "^4.44.2",
+    "webpack-dev-server": "^3.11.0"
```

Comment:
Is webpack-dev-server necessary in your dependencies?

Reply:
@rickboot Good point. I will remove it from dependencies!